### PR TITLE
[Kaptain] MLflow

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-88077

## Description of changes being made

Synced with Alex on where this topic should go. We decided to put it in the tutorials section, but not within another folder in order to make it visible. This is more like a landing intro page, since all relevant documentation will be handled upstream and in the tutorial example https://github.com/mesosphere/kaptain/pull/780#issuecomment-1172221324 .

### Preview

http://docs-d2iq-com-pr-4563.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

**Comment by juliangieseke** [comment]
seems like menu item for the new page is missing 🤔

File: pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- Gather and track data related to experiments for analysis.
+- Package ML/AI code and share it with other data scientists or transfer to production environments.
+- Manage and deploy models from ML libraries to model serving and inference platforms.
+- Manage the full lifecycle of an MLflow model from a centralized store (model versioning, stage transitions, annotations, etc.)
+
+## Prerequisites
+
+- You have launched a Jupyter notebook with Kaptain. (Using another notebook would require you to build a custom image for this purpose)
+
+  <p class="message--important"><strong>IMPORTANT: </strong>If you decide to use your own custom image or an image not provided by Kaptain to run a distributed training job, create the image following the recommended container image creation steps (<a href="https://catalog.ngc.nvidia.com/orgs/nvidia/containers/l4t-base">Running an l4t-base container</a>, <a href="https://gitlab.com/nvidia/container-images/l4t-base">l4t-base build instructions</a>). Also ensure MLflow has been installed on it.</p>
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md:24]: SGTM! Tbh I think I found those links while doing my research and assumed that was the correct way. But referencing our own docs about it is definitely much better!
```suggestion
  <p class="message--important"><strong>IMPORTANT: </strong>If you decide to use your own custom image or an image not provided by Kaptain to run a distributed training job, create an image following the (<a href="../sdk/image-builder/">recommended container image creation steps</a>. Also ensure MLflow has been installed on it.</p>
```

**Review by cneth — APPROVED** [review]

File: pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+- You have launched a Jupyter notebook with Kaptain. (Using another notebook would require you to build a custom image for this purpose)
+
+  <p class="message--important"><strong>IMPORTANT: </strong>If you decide to use your own custom image or an image not provided by Kaptain to run a distributed training job, create an image following the (<a href="../sdk/image-builder/">recommended container image creation steps</a>. Also ensure MLflow has been installed on it.</p>
+
+- Optional: If you are using SDK, you have [set up Kaptain SDK for your desired training framework](https://docs.d2iq.com/dkp/kaptain/2.0.0/tutorials/sdk/).
+
+## Set up MLflow
+
+Starting from this release, MLflow is available in every notebook per default. You do not need to download, nor install it.
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md:30]:
```suggestion
Starting with Kaptain 2.1.0, MLflow is available in every notebook per default. You do not need to download, nor install it.
```

**Review by lukeogg — APPROVED** [review]

File: pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md (hunk 2 above)
Comment (gvasquezvargas) [reply pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md:30]: We usually avoid using version numbers whenever we can, so we don't need to touch the pages between releases. However, you made me realize we should not use wordings like "starting from this release" either 🙈 
```suggestion
MLflow is available in every notebook per default. You do not need to download, nor install it.
```

File: pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Manage and deploy models from ML libraries to model serving and inference platforms.
+- Manage the full lifecycle of an MLflow model from a centralized store (model versioning, stage transitions, annotations, etc.)
+
+## Prerequisites
+
+- You have launched a Jupyter notebook with Kaptain. (Using another notebook would require you to build a custom image for this purpose)
+
+  <p class="message--important"><strong>IMPORTANT: </strong>If you decide to use your own custom image or an image not provided by Kaptain to run a distributed training job, create an image following the (<a href="../sdk/image-builder/">recommended container image creation steps</a>. Also ensure MLflow has been installed on it.</p>
+
+- Optional: If you are using SDK, you have [set up Kaptain SDK for your desired training framework](https://docs.d2iq.com/dkp/kaptain/2.0.0/tutorials/sdk/).
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md:26]:
```suggestion
- Optional: If you are using SDK, you have [set up Kaptain SDK for your desired training framework](../sdk/).
```

File: pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Use MLflow
+
+For more information on how to use MLflow Tracking, MLflow Projects, or MLflow Models, refer to the [MLflow][mlflow_docs] documentation.
+
+If you want to see an example of how to log SDK metrics in MLflow, refer to the [Log and Hyperparameters Metrics to MLFlow][tutorial] tutorial.
+
+[mlflow_docs]: https://mlflow.org/docs/1.25.1/index.html
+[tutorial]: https://docs.d2iq.com/dkp/kaptain/2.1.0/tutorials/sdk/quick-start#log-and-hyperparameters-metrics-to-mlflow
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md:45]:
```suggestion
[tutorial]: ../sdk/quick-start#log-and-hyperparameters-metrics-to-mlflow
```

**Review by mmcelroy91 — APPROVED** [review]

**Pull request merged**